pkg/framer/property_value_history.go

```diff
@@ -44,7 +44,7 @@ func (p AssetPropertyValueHistory) Frames(ctx context.Context, resources resourc
 		if v.Value != nil && getPropertyVariantValue(v.Value) != nil {
 			timeField.Set(i, getTime(v.Timestamp))
 			valueField.Set(i, getPropertyVariantValue(v.Value))
-			qualityField.Set(i, v.Quality)
+			qualityField.Set(i, string(v.Quality))
```

Comment:
The edge region doesn't use the batch api. This change follows what's done in the main v2 upgrade PR https://github.com/grafana/iot-sitewise-datasource/pull/402/files#diff-fbd40360883c5e3ae50002127791ee48cd59f80b5f4d7292f12454446a4a7308R116